package.json

```diff
@@ -17,7 +17,7 @@
         "@types/node": "^10.17.27",
         "@typescript-eslint/eslint-plugin": "^4.23.0",
         "@typescript-eslint/parser": "^4.23.0",
-        "aws-cdk": "^1.119.0",
+        "aws-cdk": "^1.122.0",
```

Comment:
Please don't update cdk version just yet. @shapirov103  usually deals with that during the release cycle.

Reply:
Done